extensions/raindrop/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "search_raindrop",
+  "title": "Search Raindrop",
+  "description": "Searches Raindrop",
```

Comment:
Considering that at the moment you only support one command, we encourage you to have a more specific extension description. For now, it can be even the same as for the command.